packages/lib/src/vis/shared/SelectionMesh.tsx

```diff
@@ -100,14 +104,13 @@ function SelectionMesh(props: Props) {
       setStartPoint(undefined);
       setEndPoint(undefined);
     },
-    [startPoint, camera, worldToData, onSelectionEnd]
+    [startPoint, size, camera, worldToData, onSelectionEnd]
   );
 
   return (
-    <VisMesh {...{ onPointerMove, onPointerUp, onPointerDown, ...meshProps }}>
-      <meshBasicMaterial opacity={0} transparent />
+    <InteractionMesh {...{ onPointerDown, onPointerMove, onPointerUp }}>
       {startPoint && endPoint ? children({ startPoint, endPoint }) : null}
-    </VisMesh>
+    </InteractionMesh>
```

Comment:
By adding a `children` prop to `InteractionMesh`, I was able to use in `SelectionMesh`